test-project/components/MainScene.brs

```diff
@@ -57,31 +57,45 @@ end function
 
 
 function configureLabel()
-  m.label.allFontSettings = {
-    header1Font: {
-      name: "font:MediumBoldSystemFont"
-      size: 130
-    }
-    header2Font: {
-      name: "font:MediumBoldSystemFont"
-      size: 60
-    }
-    header3Font: {
-      name: "font:MediumSystemFont"
-      size: 50
-    }
-    font: {
-      name: "font:MediumSystemFont"
-      size: 30
-    }
-    italicFont: {
-      name: "font:MediumBoldSystemFont"
-      size: 32
-    }
-    boldFont: {
-      name: "font:MediumSystemFont"
-      size: 34
-  } }
+  ' m.label.allFontSettings = {
```

Comment:
Why are these commented out? If they aren't needed anymore, then we should delete the code.

Reply:
Cleaned up.